vms/platformvm/service.go

```diff
@@ -1828,6 +1830,67 @@ func (s *Service) GetBlockByHeight(_ *http.Request, args *api.GetBlockByHeightAr
 	return err
 }
 
+// DynamicFeesConfigReply is the response from GetDynamicFeeConfig
+type DynamicFeesConfigReply struct {
+	commonfee.DynamicFeesConfig `json:"nextGasPrice"`
+}
+
+// GetNextFeeRates returns the next fee rates that a transaction must pay to be accepted now
+func (s *Service) GetDynamicFeeConfig(_ *http.Request, _ *struct{}, reply *DynamicFeesConfigReply) error {
```

Comment:
The method, as implemented, makes me feel a bit uncomfortable. The `reply` is being used in some cases, but not in others, and the caller is expected to deduce that by examining the error code. In addition, the caller is expected to provide a valid `reply` pointer, or the method might (sometimes) panic.

I know that this aligns with the existing calling convension in this file, and was wondering if there is a good reason for that. In particular, I would that written it as 
```golang
func (s *Service) GetDynamicFeeConfig(_ *http.Request, _ *struct{}) (reply *DynamicFeesConfigReply, err error) 
```
since that would have the convention that the `err` and the `reply` are mutually exclusive.